src/pages/Home/home.test.tsx

```diff
@@ -0,0 +1,210 @@
+          _id: "mockSubscriptionId",
+          source: "mockSubscriptionSource"
+          }
+        }]
+      }
+    render( 
+      <UserContext.Provider value={mockUserContextUpdated}>
+        <FrequencyContext.Provider value={mockFrequencyContext} >
+          <Home />
+        </FrequencyContext.Provider>
+      </UserContext.Provider> 
+      );
+      expect(screen.getByText(Enums.SUBS_LIST_TITLE)).toBeInTheDocument();
+      expect(screen.getByText(/daily/)).toBeInTheDocument();
+      expect(screen.getByText(/08:00/)).toBeInTheDocument();
+      expect(screen.getByText(/mockSubscriptionName/)).toBeInTheDocument();
+      expect(screen.getByText(/unsubscribe/)).toBeInTheDocument();
+      expect(screen.getByText(Enums.ADD_NEW_SUBS_MESSAGE)).toBeInTheDocument();
+      expect(screen.getByText(/mockSubscriptionUrl/)).toBeInTheDocument();
+      expect(screen.getByRole("button", {name: "Settings"})).toBeInTheDocument();
+      expect(screen.getByText("Mock Footer Component")).toBeInTheDocument();
+      MockDate.reset();
+  });
+
+  test('Home - user with subscriptions can open settings & interact with features', async () => {
+    MockDate.set(new Date('1/01/2023'));
+    const mockUserContextUpdated = {
+      ...mockUserContext, 
+      isDataLoading: false, 
+      subscriptions: [{
+        subscription: {
+          name: "mockSubscriptionName",
+          url: "mockSubscriptionUrl",
+          _id: "mockSubscriptionId",
+          source: "mockSubscriptionSource"
+          }
+        }]
+      }
+    render( 
+      <UserContext.Provider value={mockUserContextUpdated}>
+        <FrequencyContext.Provider value={mockFrequencyContext} >
+          <Home />
+        </FrequencyContext.Provider>
+      </UserContext.Provider> 
+      );
+      const settingsBtn = screen.getByRole("button", {name: "Settings"});
+      fireEvent.click(settingsBtn);
+      expect(screen.getByText(/Change Settings/)).toBeInTheDocument();
+      expect(screen.getByText("Summaries of digest (On)")).toBeInTheDocument();
+      const toggleSummaryBtn = screen.getByTestId("toggle-summary-btn");
+      fireEvent.click(toggleSummaryBtn);
+      expect(screen.getByText("Summaries of digest (Off)")).toBeInTheDocument();
+      const pauseDigestBtn = screen.getByText("Pause digest");
+      expect(pauseDigestBtn).toBeInTheDocument();
+      fireEvent.click(pauseDigestBtn);
+      // expect(screen.getByText("Your digest is paused, start receiving it again")).toBeInTheDocument();
+      // const resumeDigestBtn = screen.getByText("Resume digest");
+      // expect(resumeDigestBtn).toBeInTheDocument();
+      // fireEvent.click(resumeDigestBtn);
+      // expect(pauseDigestBtn).toBeInTheDocument();
+      // const closeSettingsBtn = screen.getByText("Close");
+      // expect(closeSettingsBtn).toBeInTheDocument();
+      // fireEvent.click(closeSettingsBtn);
+      // expect(screen.getByText(/Change Settings/)).not.toBeInTheDocument();
+      // MockDate.reset();
```

Comment:
Why are these commented out?

Reply:
Same click issue.
If I fireEvent click on an element and the click causes an effect that will make the element itself change the data its rendering, will that be an issue?